src/index.html

```diff
@@ -5,8 +5,299 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
```

Comment:
The link element should point to a CSS file instead of a SCSS file. Change the href value to './styles/main.css'